mnemocards/builders/autogenerate_builder.py

```diff
@@ -253,7 +253,7 @@ def build_cards_from_words(self, settings):
         words = tsv.scrape_words_from_file(settings["filename"])
         translations = tsv.get_translation(
             words, settings["lang"]["original"], settings["lang"]["translation"])
-        cards = [tsv.prepare_card_fields(trans) for trans in translations]
+        cards = [tsv.prepare_card_fields(trans) for trans in translations if tsv.prepare_card_fields(trans)]
```

Comment:
@mkXultra the problem you identified is because there are some `None` values in the list `cards` after the list comprehension, so this does fix the problem with `NoneType` error.